maze-info.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/activity/activity.info b/activity/activity.info
+index abb1d02..1bfab78 100644
+--- a/activity/activity.info
++++ b/activity/activity.info
+@@ -3,9 +3,16 @@ name = Maze
+ exec = sugar-activity3 activity.MazeActivity
+ icon = activity-maze
+ activity_version = 29
++release_date = 2018-04-12
+ license = GPLv3+
+-bundle_id = vu.lux.olpc.Maze
+-summary = Good at finding your way out of hard situations? Let's see! Try to make your way through an increasingly difficult path, or you can also play with a friend!
+-max_participants = 6
++metadata_license = CC0-1.0
++update_contact = tch@sugarlabs.org
++bundle_id = org.sugarlabs.Maze
++summary = A simple maze game for the Sugar learning environment
++max_participants = 1
+ repository = https://github.com/sugarlabs/maze-activity
+ tags = Game
++url = https://github.com/sugarlabs/maze-activity
++description = Good at finding your way out of hard situations? Let's see! Try to make your way through an increasingly difficult path.
++screenshots = https://i.imgur.com/DqZbIHp.png https://i.imgur.com/KW82dql.png https://i.imgur.com/UzxCIGe.png
```

Comment:
I ran the validator and got this:

```
$ appstream-util validate /var/lib/flatpak/app/org.sugarlabs.MazeActivity/current/active/files/share/appdata/org.sugarlabs.MazeActivity.appdata.xml 
/var/lib/flatpak/app/org.sugarlabs.MazeActivity/current/active/files/share/appdata/org.sugarlabs.MazeActivity.appdata.xml: FAILED:
• attribute-invalid     : <screenshot> height too large [https://i.imgur.com/DqZbIHp.png] maximum is 900px
• attribute-invalid     : <screenshot> height too large [https://i.imgur.com/KW82dql.png] maximum is 900px
• attribute-invalid     : <screenshot> height too large [https://i.imgur.com/UzxCIGe.png] maximum is 900px
Validation of files failed
[tchx84@silverblue pflathuv]$
```

Can you the reduce the screenshots size?

Reply:
7e1dce6